### PR TITLE
update to point to release 1.9

### PR DESCRIPTION
**Pull request by shamsher31**
As part of this step

> Update all repos that point to master branch of another repo to point to the release branch. master-latest-daily -> 1.9-dev (see https://discuss.istio.io/t/deprecation-notice-daily-builds/3991), latest -> 1.9-dev, git clone master to git clone release-1.9, etc.

Part of https://github.com/istio/istio/issues/29490#

### CODE REVIEW

**Comment by shamsher31** [comment]
@ericvn I haven't found any `master` was replaced in the previous `release-1.8` branch, so not sure If this step is needed. Please confirm.

**Comment by ericvn** [comment]
@shamsher31 I believe that there wasn't any changes from `master` in release-1.8, but @GregHanson should know what they did for sure.

File: perf/other/multicluster-vpc/setup.sh [hunk 1]
```diff
@@ -104,7 +104,7 @@ function install_istio() {
 	pushd tmp/istio-${RELEASE}
 	# shellcheck disable=SC2086
 	for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
-	helm repo add istio.io "https://storage.googleapis.com/istio-prerelease/daily-build/master-latest-daily/charts"
+	helm repo add istio.io "https://storage.googleapis.com/istio-prerelease/daily-build/release-1.9/charts"
```
Comment (howardjohn) [inline perf/other/multicluster-vpc/setup.sh:107]: this doesn't actually exist, but the whole script is defunct now so it should probably just be entirely removed

File: perf/stability/istio-chaos-partial/values.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 namespace: istio-chaos-partial
 chaosIntervalMinutes: 1
 components: istio-pilot istio-citadel istio-galley istio-policy istio-telemetry istio-tracing
-kubectlImage: gcr.io/istio-release/kubectl:master-latest-daily
+kubectlImage: gcr.io/istio-release/kubectl:release-1.9
```
Comment (howardjohn) [inline perf/stability/istio-chaos-partial/values.yaml:4]: This doesn't exist

**Comment by shamsher31** [comment]
In that case not sure if this step is valid or needed from https://github.com/istio/istio/issues/29490

> Update all repos that point to master branch of another repo to point to the release branch. master-latest-daily -> 1.9-dev (see https://discuss.istio.io/t/deprecation-notice-daily-builds/3991), latest -> 1.9-dev, git clone master to git clone release-1.9, etc

**Comment by jacob-delgado** [comment]
Let's close this out, but we can clean up a few of the issues John pointed out in master and cherry-pick back into release-1.9 if we desire.

**Pull request closed without merging**